### DISCUSSION TITLE
Was told to come here

### DISCUSSION THREAD

**Discussion by Roxyredcat** [ant-media/Ant-Media-Server#5785] [General]
Hello,

I was told to come here and follow the instructions to receive assistance on an MP4 file that is not uploading. Not sure what else to do.

**Comment by tremzo** [comment]
@Roxyredcat can you detail the exact issue and if there are any errors? Can you provide your error/server log so we can check? When you say it's not uploading, do you mean the web panel refuses to upload the file or it tries uploading and it never finishes the process? 

If you can provide some logs or screenshots we will be able to try and identify the issue.

**Comment by Roxyredcat** [comment]
I am pretty certain that the files I've been dealing with are Mp3 and Mp4.
I tried to convert the Mp3 to Mp4, but I don't think it has been converting
properly. The system will not accept the file as an Mp4, even though I told
Windows Movie Maker to convert it to Mp4 when exporting. Then I tried
another way. I went online and tried to convert the Mp3 alone to an Mp4.
Then I put it into Windows Movie Maker (Because that's what I've been using
to make my file to upload to YouTube), and it didn't play the sound. So I'm
at a complete loss as to what to do. This whole thing has been nothing but
a stressful mess, and I've spent hours trying to get it to work.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Roxyredcat** [comment]
I'm unable to provide my file or screenshots at this moment because I'm on my phone and am logged into my work systems. So I will have to do that later.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Roxyredcat** [comment]
Hello,

Sorry it took so long to respond. I think the web panel refuses to upload the file. Here is a screenshot.

The Mp4 file is also attached. The sound is from Mp3...I already converted the sound to Mp4 and tried to add the sound into Windows Movie Maker, but it won't play the sound. The Mp4 music plays without Windows Movie Maker, but it won't play on Movie Maker. So I don't know what else to do. [https://res.cdn.office.net/assets/mail/file-icon/png/video_16x16.png] Rainy Days & Tea Time.mp4<https://1drv.ms/v/s!Aoax8ogcQgPgkj1S6QHgkAPQVtoa>

Thank you 🙂
Roxanne Carter
________________________________

**Reply by tremzo** [reply]
Please email me with your web panel credentials and I can take a look tim.bobker@antmedia.io. The screenshot is loading a 16x16 image, I think you must've added the wrong image file to your response.

**Comment by Roxyredcat** [comment]
When you say "web panel credentials" does that mean my login so you are able to look in my account, or are you talking about the Logs? I am new to Ant Media.
________________________________

**Comment by mekya** [comment]
Hi @Roxyredcat and @timantmedia 
Maybe a quick meeting can save good amount of time. 

What's your availability @Roxyredcat for the next days?

**Comment by tremzo** [comment]
@Roxyredcat @mekya is correct, this needs to be resolved over a meeting. Please let me know your availability and we can connect and resolve the issue.

**Comment by Roxyredcat** [comment]
Good morning,

Are you guys available after 6:30pm EST this evening? If not, it will have to be Friday because I am off. Otherwise I work until 6:30.

Thank you!
Roxanne Carter
________________________________

**Comment by tremzo** [comment]
@Roxyredcat It's a bit late in GMT (23.30), let's schedule for Friday. I'll be able available until 21:00 GMT / 16:00 EST.  Please send me an email at tim.bobker@antmedia.io and we can schedule the meeting.
